Tests/get_build_content_artifacts.py

```diff
@@ -0,0 +1,159 @@
+
+    Arguments:
+        content_zipfile_name: (str)
+            The name of the content zipfile whose content-descriptor.json file needs modification
+
+    Returns:
+        (str)
+        The name of the (updated) content zipfile
+    '''
+    zipfile_name = content_zipfile_name
+    zipfile_write = '.'.join(zipfile_name.split('.')[:-1]) + 'modded.zip'
+    if os.path.isfile(zipfile_write):
+        os.remove(zipfile_write)
+    content_descriptor = 'content-descriptor.json'
+    zipwriter = ZipFile(zipfile_write, 'w', compression=ZIP_DEFLATED)
+    directory = './extracted_content'
+    print('extracting files from content zipfile "{}" to "{}" directory'.format(zipfile_name, directory))
+    with ZipFile(zipfile_name, 'r') as the_zip:
+        if os.path.exists(directory):
+            shutil.rmtree(directory)
+        os.makedirs(directory)
+        the_zip.extractall(path=directory)
+
+    desc_json = {}
+    prev_dir = os.getcwd()
+    os.chdir(directory)
+    file_paths = get_all_file_paths('.')
+    for file_path in file_paths:
+        file_name = file_path.split('/')[-1]
+        if file_name == content_descriptor:
+            with open(file_path, 'r') as the_file:
+                file_contents = the_file.read()
+                desc_json = json.loads(file_contents)
+                release_num = desc_json.get('release')
+                print('release number was "{}"'.format(release_num))
+                split_release_num = release_num.split('.')
+                decreased_digit = str(int(split_release_num[0]) - 1)
+                new_release_num_arr = [decreased_digit]
+                new_release_num_arr.extend(split_release_num[1:])
+                new_release_num = '.'.join(new_release_num_arr)
+                print('decreaseing release number to "{}"'.format(new_release_num))
+                desc_json['release'] = new_release_num
+            with open(file_path, 'w') as the_file:
+                the_file.write(json.dumps(desc_json))
+        zipwriter.write(file_path)
+    print('zipped to new zip file "{}"'.format(zipfile_write))
+    os.chdir(prev_dir)
+    # remove original content zipfile
+    os.remove(zipfile_name)
+    # rename modified content zipfile to original content zipfile name
+    os.rename(zipfile_write, zipfile_name)
+    return zipfile_name
+
+
+def get_latest_artifacts(branch):
+    try:
+        print('Making request to get the artifacts from the latest successful build on branch "{}"'.format(branch))
+        params = {
+            'branch': branch,
+            'filter': 'successful'
+        }
+        res = requests.get('https://circleci.com/api/v1.1/project/github/demisto/content/latest/artifacts',
+                           params=params)
+        if res.status_code < 200 or res.status_code >= 300:
+            msg = 'requests exception: [{}] - ' \
+                  '{}\n{}'.format(res.status_code, res.reason, res.text)
+            raise Exception(msg)
```

Comment:
who catches that exception? do we `print_error` ?

Reply:
it happens 3 lines below but I'm actually going to remove the `try` `excepts` because if the requests fail I want the build step to fail